### PR TITLE
TC_01.004.03 | Delete a project from the Dashboard page

### PR DESCRIPTION
**Pull request by Olesia27**
https://github.com/orgs/RedRoverSchool/projects/4/views/1?filterQuery=01.004+&pane=issue&itemId=87919355&issue=RedRoverSchool%7CJenkinsQA_JS_2024_fall%7C200

### CODE REVIEW

**Review by AnDerova — APPROVED** [review]

File: cypress/e2e/freestyleProjectDeleteProject.cy.js [hunk 1]
```diff
@@ -46,4 +46,38 @@ describe('US_01.004 | FreestyleProject > Delete Project', ()=>{
...
+    it('TC_01.004.03 | Delete a project from the Dashboard page', () => {
+
+        let oldName = 'OurPapka';
+        const locateNewItemlink = '[href="/view/all/newJob"]';
+        const newItemInputfield = '#name';
+        const labelFolder= 'label';
+        const okButton = '#ok-button'; 
+        const saveButton = '[name="Submit"]';
+        const dashboardButton = '.model-link';
+        const folderNameonPanel = 'span';
```
Comment (Stavolg) [inline cypress/e2e/freestyleProjectDeleteProject.cy.js:60]: folderNameOnPanel

File: cypress/e2e/freestyleProjectDeleteProject.cy.js [hunk 2]
```diff
@@ -46,4 +46,38 @@ describe('US_01.004 | FreestyleProject > Delete Project', ()=>{
...
+        const checkMark = `[data-href="http://localhost:8080/job/${oldName}/"]`;
+        const deleteFolderElement = `[href="/job/${oldName}/doDelete"]`;
+        const yeSButton = '[data-id="ok"]'
```
Comment (Stavolg) [inline cypress/e2e/freestyleProjectDeleteProject.cy.js:63]: add ; in the end of the row

Comment (Stavolg) [inline cypress/e2e/freestyleProjectDeleteProject.cy.js:63]: yesButton (but maybe is not a good conts name as well)

File: cypress/e2e/freestyleProjectDeleteProject.cy.js [hunk 3]
```diff
@@ -46,4 +46,38 @@ describe('US_01.004 | FreestyleProject > Delete Project', ()=>{
         cy.contains('Project').should('not.exist')
         
     })
+
+
+    it('TC_01.004.03 | Delete a project from the Dashboard page', () => {
+
+        let oldName = 'OurPapka';
+        const locateNewItemlink = '[href="/view/all/newJob"]';
+        const newItemInputfield = '#name';
```
Comment (Stavolg) [inline cypress/e2e/freestyleProjectDeleteProject.cy.js:55]: newItemInputField

**Review by Stavolg — APPROVED** [review]
A lot of assertions in your TC. It might be better to create more TC cards for having only one assertions in the TC.

File: cypress/e2e/freestyleProjectDeleteProject.cy.js (hunk 2 above)
Comment (Olesia27) [reply cypress/e2e/freestyleProjectDeleteProject.cy.js:63]: I do agree, but they said variable yesButton already exists.

**Pull request merged**